### PR TITLE
Fix issues with the bell

### PR DESCRIPTION
**Pull request by Sinestia**
## What Does This PR Do
1. The bell must now be secured to be rung, you can secure it with a wrench.
2. You will no longer be ringing it in your backpack (big oops that I didn't test and notice that.)
3. Maps have been updated to reflect the anchoring requirement.

## Why It's Good For The Game
Being able to ring something in your backpack is horrible
It also means you can't just drag it through the hallway spamming it to be a major pest. 

## Images of changes
Probably not relevant?

## Testing
I went in-game, rang it to see that it was anchored by default, unwrenched it, tested it, picked it up, tried to secure it and ring it in my bags. 

## Changelog
:cl:Froststahr
tweak: The desk bell must now be anchored to be rung, you can anchor it with a wrench. The map-placed bells are anchored by default.
fix: You can no linger ring the bell from your backpack. 
/:cl:

### CODE REVIEW

File: code/modules/paperwork/desk_bell.dm [hunk 1]
```diff
@@ -63,15 +57,29 @@
...
-		to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
-		playsound(user, 'sound/items/deconstruct.ogg', 50, vary = TRUE)
-		new /obj/item/stack/sheet/metal(drop_location(), 2)
-		qdel(src)
-		return TRUE
+	. = TRUE
+	if(user.a_intent == INTENT_HARM && !in_inventory)
+		to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
+		if(tool.use_tool(src, user, 5 SECONDS))
+			to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
```
Comment (lewcc) [inline code/modules/paperwork/desk_bell.dm:66]: Probably don't want ellipses on the completion message.
```suggestion
			to_chat(user, "<span class='notice'>You disassemble the bell.</span>")
```

File: code/modules/paperwork/desk_bell.dm [hunk 2]
```diff
@@ -63,15 +57,29 @@
...
-	to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
-	if(tool.use_tool(src, user, 5 SECONDS))
-		to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
-		playsound(user, 'sound/items/deconstruct.ogg', 50, vary = TRUE)
-		new /obj/item/stack/sheet/metal(drop_location(), 2)
-		qdel(src)
-		return TRUE
+	. = TRUE
+	if(user.a_intent == INTENT_HARM && !in_inventory)
+		to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
```
Comment (lewcc) [inline code/modules/paperwork/desk_bell.dm:64]: Might want to make these visible messages instead so other people can see them.
```suggestion
		visible_message("<span class='notice'>[user] begins taking apart the bell...</span>", "<span class='notice'>You begin taking apart the bell</span>")
```

File: code/modules/paperwork/desk_bell.dm (hunk 1 above)
Comment (lewcc) [reply code/modules/paperwork/desk_bell.dm:66]: Might also want to add some completion messages for the other tool actions while you're at it

File: code/modules/paperwork/desk_bell.dm (hunk 2 above)
Comment (Sinestia) [reply code/modules/paperwork/desk_bell.dm:64]: I believe most of these only report it on completion, with the sound being the indicator that something is happening

Comment (Sinestia) [reply code/modules/paperwork/desk_bell.dm:64]: Actually, I'm probably wrong here

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Sinestia** [comment]
Hold on, spotted string consistency issues

**Comment by Sinestia** [comment]
Okay NOW it should be good

**Comment by hal9000PR** [comment]
the sci remap accidentally removed the bells from science. would you readd them?

**Comment by Sinestia** [comment]
> the sci remap accidentally removed the bells from science. would you readd them?

Done

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**